doc/en/mooncake-store.md

```diff
@@ -471,7 +471,9 @@ Currently, an approximate LRU policy is adopted, where the least recently used o
 
 ### Lease
 
-To avoid data conflicts, a per-object lease will be granted whenever an `ExistKey` request or a `GetReplicaListRequest` request succeeds. An object is guaranteed to be protected from `Remove` request, `RemoveAll` request and `Eviction` task until its lease expires. A `Remove` request on a leased object will fail. A `RemoveAll` request will only remove objects without a lease.
+To avoid data conflicts, a per-object lease is granted whenever an `ExistKey` request or a `GetReplicaListRequest` request succeeds. While the lease is active, the object is protected from `Remove`, `RemoveAll`, and `Eviction` operations. Specifically, a `Remove` request targeting a leased object will fail, and a `RemoveAll` request will only delete objects without an active lease. This ensures that the object’s data can be safely read as long as the lease has not expired.
```

Comment:
just a suggestion, maybe we could write "during lease, the object is read-only (or immutable)"

Reply:
The immutability of an object once it is put is a fundamental property of the store, not limited to the lease period. Let me introduce this feature in another section.